docs/tables/aws_pinpoint_app.md

```diff
@@ -0,0 +1,55 @@
+# Table: aws_pinpoint_app
+
+In Amazon Pinpoint, a project is a collection of recipient information, segments, campaigns, and journeys. New Amazon Pinpoint users should start by creating a project.
```

Comment:
The description talks about a project instead of an application(app). Could we add better description around the app?

Reply:
@misraved In doc it is mentioned that the project is same as the application there is no other resource type as application. https://docs.aws.amazon.com/pinpoint/latest/userguide/gettingstarted-create-project.html

Reply:
@ParthaI Amazon Pinpoint is an AWS service that you can use to engage with your customers across multiple messaging channels. You can use Amazon Pinpoint to send push notifications, in-app notifications, emails, text messages, voice messages, and messages over custom channels. It includes segmentation, campaign, and journey features that help you send the right message to the right customer at the right time over the right channel.